web/app/types/project.d.ts

```diff
@@ -2,22 +2,26 @@ import {
   RelatedExternalLink,
   RelatedHermesDocument,
 } from "hermes/components/related-resources";
+import { ProjectStatus } from "./project-status";
+
+export interface JiraObject {
+  key: string;
+  url: string;
+  priority: string;
+  status: string;
+  assignee?: string;
+  type?: string;
```

Comment:
Is there a case where Jira objects won't have a type?

Reply:
Yeah probably not. Updated.